ui/app/[locale]/(authorized)/kafka/[kafkaId]/consumer-groups/[groupId]/reset-offset/ResetOffset.tsx

```diff
@@ -215,6 +212,18 @@ export function ResetOffset({
                       type="number"
                       min={0}
                     />
+                    {error && (
```

Comment:
My only concern here is that `error` could be other things besides the custom offset being out of range. The other options are reporting an error from Kafka or that a requested partition is not valid for the topic. I think those other options are much less likely than an out-of-range error, but just to mention that we would be masking them as an out-of-range error.